From 8545129540a5862b22aad03badb2a9f93bf29117 Mon Sep 17 00:00:00 2001
From: Bob Cochran <yocto@mindchasers.com>
Date: Mon, 3 Nov 2014 22:45:35 -0500
Subject: [meta-fsl-ppc][PATCH] linux-qoriq: Change defconfig for T1040 to
 match number of CPUS

Having a number higher than necessary for NR_CPUS wastes memory by
instantiating unnecessary structures in RAM.  An example is in the DPAA where
DPAA_ETH_TX_QUEUES is defined based on NR_CPUS and used to create
dozens of extra qman_fq structures.  Using the prior value of 24, which was
left over from the T4240 created an additonal 60 frame queue structures alone.

This has been tested on t1040rdb-64b.  .

Signed-off-by: Bob Cochran <yocto@mindchasers.com>
---
 arch/powerpc/configs/corenet32_fmanv3l_smp_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/configs/corenet32_fmanv3l_smp_defconfig b/arch/powerpc/configs/corenet32_fmanv3l_smp_defconfig
index a401e7c..5542248 100644
--- a/arch/powerpc/configs/corenet32_fmanv3l_smp_defconfig
+++ b/arch/powerpc/configs/corenet32_fmanv3l_smp_defconfig
@@ -1,6 +1,6 @@
 CONFIG_PPC_85xx=y
 CONFIG_SMP=y
-CONFIG_NR_CPUS=8
+CONFIG_NR_CPUS=4
 CONFIG_EXPERIMENTAL=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
--
1.7.9.5
